eng/Versions.props

```diff
@@ -25,12 +25,11 @@
     <MicrosoftVisualStudioExtensibilityTestingVersion>0.1.127-beta</MicrosoftVisualStudioExtensibilityTestingVersion>
     <!-- CodeStyleAnalyzerVersion should we updated together with version of dotnet-format in dotnet-tools.json -->
     <CodeStyleAnalyzerVersion>4.0.1</CodeStyleAnalyzerVersion>
-    <VisualStudioEditorPackagesVersion>16.10.230</VisualStudioEditorPackagesVersion>
-    <VisualStudioEditorNewPackagesVersion>17.0.487</VisualStudioEditorNewPackagesVersion>
+    <VisualStudioEditorNewPackagesVersion>17.2.178-preview</VisualStudioEditorNewPackagesVersion>
```

Comment:
Should we be renaming this to remove the "New"?

Reply:
Yes, I was just trying to keep the diff small. Happy to change this.

Reply:
Fixed.